tests/llama_stack/rag/test_rag.py

```diff
@@ -17,7 +18,7 @@
     "unprivileged_model_namespace, llama_stack_server_config",
```

Comment:
How about adding a boolean fixture param `randomize_name` to the fixture `unprivileged_model_namespace` and add the random part there?